fastcrypto/src/secp256r1.rs

```diff
@@ -0,0 +1,492 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//! This module contains an implementation of the [ECDSA signature scheme](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm) over the
+//! [secp256r1 NIST-P1 curve](https://www.secg.org/SEC2-Ver-1.0.pdf).
+//!
```

Comment:
I think we need a bit more precision here on:
1. whether the reader can find an implementation of RFC 6979 in the below,
2. whether they should expect to find one in the future (forward guidance)

Reply:
It does use an implementation of RFC6979 (from the p256 crate). I've left a comment on this.